tutoraspects/patches/local-docker-compose-services

```diff
@@ -123,3 +121,31 @@ aspects-consumer:
     {% if RUN_MONGODB %}- mongodb{% endif %}
     {% if RUN_REDIS %}- redis{% endif %}
 {% endif %}
+
+{% if RUN_ASPECTS_DOCS %}
+aspects-docs:
+  user: root
+  image: {{ DOCKER_IMAGE_ASPECTS }}
+  environment:
+    VENV_DIR: /opt/venv
+    ASPECTS_EVENT_SINK_DATABASE: {{ASPECTS_EVENT_SINK_DATABASE}}
+    ASPECTS_XAPI_DATABASE: {{ASPECTS_XAPI_DATABASE}}
+    CLICKHOUSE_CLUSTER_NAME: {{CLICKHOUSE_CLUSTER_NAME}}
+    DBT_STATE: {{ DBT_STATE_DIR }}
+    ASPECTS_DATA_TTL_EXPRESSION: "{{ ASPECTS_DATA_TTL_EXPRESSION }}"
+    DBT_PROFILE_TARGET_DATABASE: "{{ DBT_PROFILE_TARGET_DATABASE }}"
+    DBT_PROFILES_DIR: /app/aspects/dbt/
+    DBT_BRANCH: "{{ DBT_BRANCH }}"
+    DBT_REPOSITORY: "{{ DBT_REPOSITORY }}"
+    DBT_SSH_KEY: "{{ DBT_SSH_KEY }}"
+  command: |
+    bash /app/aspects/scripts/docs.sh
+  ports:
+    - 7000:7000
+  volumes:
+    - ../../env/plugins/aspects/apps/aspects/dbt:/app/aspects/dbt
+    - ../../env/plugins/aspects/apps/aspects/scripts:/app/aspects/scripts
```

Comment:
reminder: this volume is duplicated, and needs to be removed.

Reply:
Does this still need to be done?